tests/unit/tags/test_tag.py

```diff
@@ -13,15 +13,15 @@
 # limitations under the License.
 
 import unittest
-from opencensus.tags import Tag, TagKey, TagValue
+from opencensus.tags import Tag
 
 
 class TestTag(unittest.TestCase):
 
     def test_constructor(self):
         key = 'key1'
         value = 'value1'
-        tag = Tag(key=TagKey(key), value=TagValue(value))
+        tag = Tag(key=key, value=value)
```

Comment:
Suggestion: consider adding tests on invalid strings as key and value when creating a tag.

Reply:
Good point.

I've added the following two tests just in case the string didn't go through TagKey/TagValue constructor, rely on test_tag_key and test_tag_value for comprehensive tests:
```
        self.assertRaises(ValueError, lambda: Tag(key='', value=value))
        self.assertRaises(ValueError, lambda: Tag(key=key, value='\0'))
```